features/collection/shell-commands/src/main/java/org/opennms/netmgt/collection/commands/CollectCommand.java

```diff
@@ -77,6 +85,9 @@
     @Option(name = "-n", aliases = "--node", description = "Node ID or FS:FID", required = false, multiValued = false)
     String nodeCriteria;
 
+    @Option(name = "--persist", description = "Persist collection")
```

Comment:
Since all the other options have both a name and an alias, should we not do the same here? `-p` seems like a natural fit.

Reply:
yeah, I mistook that boolean option can only have name. It's not.